app/controllers/Application.java

```diff
@@ -607,13 +616,8 @@ private static Result getFlowHistory(Version version) {
         return ok(flowHistoryPage.render(flowDefPair.getId(), graphType,
             flowHistoryResults.render(flowDefPair, executionMap, idPairToJobNameMap, flowExecTimeList)));
       } else if (graphType.equals("resources") || graphType.equals("time")) {
-        if (hasSparkJob) {
```

Comment:
Why is this removed? Is it no longer required? If so, can you also remove the variable and its computation above in this file?